### PR TITLE
support .sass files

### PR DESCRIPTION
**Pull request by FiberJW**
updated the regex to include `.sass` files as well

### CODE REVIEW

**Comment by eanplatter** [comment]
Would it be trivial to add a `.sass` file in the example directory? Just to have a sort of test? If it's easy let's do it.

File: webpack.config.build.js [hunk 1]
```diff
@@ -40,7 +40,11 @@ module.exports = {
         loader: 'json'
       },
       {
-        test: /\.[s]?css$/,
+        test: /\.(otf|eot|ttf|woff)/,
+        loader: 'url-loader?limit=8192'
```
Comment (eanplatter) [inline webpack.config.build.js:44]: Oh good catch!

**Comment by FiberJW** [comment]
Alrighty @eanplatter

**Comment by FiberJW** [comment]
Darn. The sass loader doesn't actually support actual sass syntax. just scss

**Comment by FiberJW** [comment]
I'll make another pr for the url loader fix though

**Comment by eanplatter** [comment]
kk I'll try to figure out what's going on with sass

**Comment by FiberJW** [comment]
@eanplatter the loader just doesn't support it. Just SCSS

**Pull request closed without merging**